practice1/stylesheet.css

```diff
@@ -75,15 +75,75 @@ hr {
 }
 .blocks-layout {
     display: grid;
-    grid-template-rows: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr;
     grid-template-columns: 1fr 1fr 1fr;
     grid-gap: 1vw;
+    margin-bottom: 62px;
+}
+.shop-btn-container {
+    display: flex;
+    align-items: center;
+    gap: 8px;
 }
 .img-text {
     position: relative;
 }
 .img-text h4 {
-    position: absolute;
     margin-top: 12px;
     margin-left: 12px;
+}
+.caption-favorite {
+    display: flex;
+    position: absolute;
+    width: 450px;
+    justify-content: space-between;
+}
+.caption-favorite img {
+    align-self: center;
+    margin-right: 15px;
+    image-resolution: snap;
+}
+.block-caption {
+    display: flex;
+}
+.block-caption a {
+    margin-right: 188px;
```

Comment:
You're offsetting a sibling element with this hard-code margin, which is neither pretty, nor convenient.
You can either leave `.block-caption` flex, and apply equal `flex-grow` to both its children so they take up equal space (not 100% sure, but would probably work). Or just set both elements to 50% width.